db/migrate/20150706222818_create_messages.rb

```diff
@@ -0,0 +1,19 @@
+class CreateMessages < ActiveRecord::Migration
+  def change
+    create_table :messages do |t|
+      # http://stackoverflow.com/questions/1592291/what-is-the-email-subject-length-limit
```

Comment:
This should probably be moved into message actually since we will eventually squash this.